sklearn/ensemble/gradient_boosting.py

```diff
@@ -931,12 +931,14 @@ def _resize_state(self):
             raise ValueError('resize with smaller n_estimators %d < %d' %
                              (total_n_estimators, self.estimators_[0]))
 
-        self.estimators_.resize((total_n_estimators, self.loss_.K))
-        self.train_score_.resize(total_n_estimators)
+        self.estimators_ = np.resize(self.estimators_,
+                                     (total_n_estimators, self.loss_.K))
```

Comment:
I'm not sure the difference here between [`numpy.ndarray.resize`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.resize.html#numpy.ndarray.resize) and [`numpy.resize`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.resize.html) matters, particularly with respect to performance and

> [numpy.resize docstring]
> If the new array is larger than the original array, then the new array is filled with repeated copies of a. Note that this behavior is different from a.resize(new_shape) which fills with zeros instead of repeated copies of a.

Maybe @amueller or @glemaitre would have an opinion?

`ndarray.reshape` is not supported on PyPy..

Reply:
Thanks for the insightful comment!

> Your comment mixes up reshape and resize

Yes, definitely I meant `resize` in my comment (edited it). Though